tests/genrule/BUILD.bazel

```diff
@@ -0,0 +1,15 @@
+load("//helm:defs.bzl", "helm_chart")
+
+helm_chart(
+    name = "chart",
+    chart = "Chart.yaml",
+    values = "values.yaml",
+)
+
+genrule(
+    name = "genrule",
+    srcs = [":chart"],
+    outs = ["template.yaml"],
```

Comment:
This file is not used by the chart above, right? What happens if you try to include it? Does using generated files not work?

Reply:
It's not, and it's not intended to be used in the chart. It just demonstrates that we can call `helm template` using the genule, which makes the resulting k8s yaml manifest for the chart + values.